[18.0][MIG] account_asset_management: Migration to version 18.0

carlos-lopez-tecnativa opened the pull request:
Depends on:
- [x] https://github.com/OCA/reporting-engine/pull/958

- Update translation from `_()` to `self.env._()`.
- Update domain on fields related to `account.account`, using `check_company=True` and `_check_company_domain = models.check_company_domain_parent_of` to support company branches.
- Update domain in record rules; the fields are required, so `company=False` is unnecessary.

TT54054
@Tecnativa @pedrobaeza @victoralmau @carlosdauden could you please review this?.

rdeheele committed:
[FIX] test floats with assertAlmostEqual

[UPD] add places arg in assertAlmostEqual tests

[UPD] flake clean

lepistone committed:
move account_asset_management* out of __unported__

add asset management modules

asset mgt update

redo

synch asset mgt with recent V7 changes

lepistone committed:
make *asset* installable again

Commit:
[FIX] Inherited model is a transient model

sbidoul committed:
[IMP] account_asset_management: use reload_on_button instead of reloading page

Commit:
[IMP][account_asset_management] Define a list of move's fields that can't be modified if a move is linked with a depriciation line

[IMP][account_asset_management] Define FIELDS_AFFETCS_ASSET_MOVE as a set directly

luc-demeyer committed:
[IMP] account_asset_management: code cosmetics

luc-demeyer committed:
account_asset_management: group invoice lines fix

luc-demeyer committed:
analytic enhancement

Commit:
[ADD][account_asset_management] Add check on account move line

Commit:
[FIX][account_asset_management] Add items on affects fields list

sbidoul committed:
[FIX] account_asset_management: do not propose special and closed periods for asset posting

Commit:
[FIX][account_asset_management] Avoid to select closed or cancelled analytic account

sbidoul committed:
[UPD] prefix versions with 8.0

luc-demeyer committed:
[10.0][MIG]account_asset_management suite refactoring + 10.0 port

grindtildeath committed:
[FIX+IMP] account_asset_management: 2 things:

* Fix compute methods dependencies and small optimizations
* Rename demo file to test and move it into the right folder

luc-demeyer committed:
[FIX] account_asset_management: Test data + onchange

grindtildeath committed:
[MIG] account_asset_management: Migration to 11.0

sbidoul committed:
[IMP] asset mgt: use exclude manifest key

This is now supported natively by Odoo 11.

norlinhenrik committed:
[MIG] account_asset_management: Migration to 12.0

MaxyMoos committed:
account_asset_management: Add unit test for invoices with multiple assets

MaxyMoos committed:
Included PR #828 Fix computation of depreciation lines when having multiple assets in invoice

sbejaoui committed:
Finish 12.0 migration of account_asset_management

* account_asset: Do not loop on all the lines to search for one linked asset

Before this change, the use of `mapped` on self did loop on all the move
lines that are included in self to get the assets, what could be very
costly for a simple write on a lot of move lines. As the goal is to raise
an error only if at least one move is linked to an asset, we break the
loop if the condition is fulfilled.

* performance improvement

* [RMV] - Remove useless dependency

In 12.0 account_fiscal_year is a standard feature no need to depend on oca
module account_fiscal_year

norlinhenrik committed:
[12.0][FIX] account_asset_management: date_range -> account_fiscal_year, account_analytic_id without domain

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 40.1% (79 of 197 strings)

Translation: account-financial-tools-12.0/account-financial-tools-12.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-12-0/account-financial-tools-12-0-account_asset_management/pt_BR/

Saran440 committed:
[12.0][IMP] account_asset_management

svalaeys committed:
Translated using Weblate (French)

Currently translated at 100.0% (195 of 195 strings)

Translation: account-financial-tools-12.0/account-financial-tools-12.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-12-0/account-financial-tools-12-0-account_asset_management/fr/

josepmy committed:
Translated using Weblate (Spanish)

Currently translated at 91.8% (179 of 195 strings)

Translation: account-financial-tools-12.0/account-financial-tools-12.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-12-0/account-financial-tools-12-0-account_asset_management/es/

pedrobaeza committed:
[FIX] account_asset_management: Don't depend on account.fiscal.year created records

* There's no need of forcing to create such records if your fiscal years are regular ones.
* FY date range computation duplicates code that is already on core.
* DummyFY is a good idea, but was not used at all. Now it is.

pedrobaeza committed:
[FIX] account_asset_management: Provide hook _compute_depreciation_amount_per_fiscal_year

This way, other method_time mechanism like localization ones is able to overwrite it.

You also got an error of variable `number` not set due to previous code. Although no
alternate implementation for a new method_time, now you don't have any error,
fallbacking to standard proportional repartition.

pedrobaeza committed:
[FIX] account_asset_management: Invisible conditions must be independent from extra method_time values

If not, added methods won't hide these fields.

luc-demeyer committed:
[FIX][12.0]asset_management - fix date string compare

norlinhenrik committed:
[MIG] 11.0 account_asset to 12.0 account_asset_management

norlinhenrik committed:
[IMP] removed type, parent_id, parent_path, child_ids from account.asset & account.asset.profile

pedrobaeza committed:
[MIG+FIX+IMP] account_asset_management: Rework migration scripts + make asset group m2m

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (193 of 193 strings)

Translation: account-financial-tools-12.0/account-financial-tools-12.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-12-0/account-financial-tools-12-0-account_asset_management/pt_BR/

luc-demeyer committed:
[IMP] account_asset_management: Add reporting

Saran440 committed:
[ADD] options 'use_leap_years'

Commit:
[ADD] allows setting a depreciation ending date

Commit:
[12.0][FIX][account_asset_management] local variable 'asset' referenced before assignment

tools/account_asset_management/models/account_asset.py", line 970, in _compute_entries
    % (asset.name, asset.code) or asset.name
UnboundLocalError: local variable 'asset' referenced before assignment

pedrobaeza committed:
[MIG] account_asset_management: Handle account_asset_disposal migration

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (200 of 200 strings)

Translation: account-financial-tools-12.0/account-financial-tools-12.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-12-0/account-financial-tools-12-0-account_asset_management/pt_BR/

pedrobaeza committed:
[MIG] account_asset_management: Handle account_asset_analytic migration

Only field renaming is needed.

pedrobaeza committed:
[FIX] account_asset_management: Fix tests on leap year

It's bad idea to let dates to be variable.

luc-demeyer committed:
[12.0][FIX]traceback when deviating FY

svalaeys committed:
Fixed dynamic build of the name of the view to be called when errors are found

HviorForgeFlow committed:
[12.0] account_financial_tools: Lint fixing

ernestotejeda committed:
[IMP] account_asset_management: black, isort

ernestotejeda committed:
[MIG] account_asset_management: Migration to 13.0

pedrobaeza committed:
[FIX] account_asset_management: Avoid error

Steps to reproduce the problem:

* Go to assets view
* Group by profile
* Unfold a group and click on an asset
* Click on "Journal Entries" smart-button
* Go back to the asset list
* Click again on the same asset (or another).
* Click on "Journal Entries" smart-button

Current behavior:

Error saying "KeyError: 'profile_id'"

Expected behavior:

No error

The cause for this is that Odoo stores in the context the key `group_by` with the
value `profile_id` in the specified chain of steps. That context entry is used for
grouping records in the list, and system tries to group the journal entries also
by that field, which doesn't exists in the other model, and thus the error.

We avoided it copying the context to be passes and leaving out that entry.

pedrobaeza committed:
[IMP] account_asset_management: Simplify code for opening related entries

We don't need the search having the one2many field and no need also for list
comprehension having mapped operator.

carlosdauden committed:
[IMP] account_asset_management: Improve code and translation

Commit:
[FIX] account_asset_management: Fix tests

qgroulard committed:
[IMP] account_asset_management: Depreciation board compute upon asset confirmation

astirpe committed:
Fix of pre-commit: E741 ambiguous variable name 'l'

ValentinVinagre committed:
[IMP] account_asset_management: Add scheduled action for the depreciation asset entries creation.

ValentinVinagre committed:
[IMP] account_asset_management: Add chatter entry in invoice for generated asset

ValentinVinagre committed:
[FIX] account_asset_management: FIX empty asset reference

If the assets are created before posting the invoice, they wouldn't have a name and the "code" field of the assets will be empty, but they would have the invoice name.

Commit:
[FIX] account_asset_management: fix remaining_value cache miss error.

HaraldPanten committed:
Translated using Weblate (Spanish)

Currently translated at 87.7% (171 of 195 strings)

Translation: account-financial-tools-13.0/account-financial-tools-13.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-13-0/account-financial-tools-13-0-account_asset_management/es/

FernandoRomera committed:
[FIX] account_asset_management:
* multicompany error

FernandoRomera committed:
[FIX] version

[FIX] new line

[FIX] new line

[FIX] without openupgrade

[FIX] without openupgrade

[FIX] without openupgrade

[FIX] without openupgrade

[FIX] without openupgrade

Saran440 committed:
[FIX] account_asset_management - Allow editing salvage value

pedrobaeza committed:
[IMP] account_asset_management: Refinements

dong-z committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 39.9% (77 of 193 strings)

Translation: account-financial-tools-13.0/account-financial-tools-13.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-13-0/account-financial-tools-13-0-account_asset_management/zh_CN/

Saran440 committed:
[FIX] calculate days when remove asset

fmdl committed:
[FIX] account_asset_management : empty message is posted if there are no asset

pedrobaeza committed:
[FIX] account_asset_management: Proper asset unlink + move unlink

2 related fixes:

- When removing an asset line depreciation move, we have to pass it
  first to draft, or we won't be able to remove it even with the
  context.
- When removing a move, the check for removing the linked asset should
  be only for purchase documents, not for "not sale" documents.

joao-p-marques committed:
[MIG] account_asset_management: Migration to v14

TT26410

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 59.0% (114 of 193 strings)

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/nl/

pedrobaeza committed:
[FIX] account_asset_management: Missing field renaming

type > move_type, due to uncovered code.

StefanRijnhart committed:
[FIX] account_asset_management: don't set class attributes in reporting

Also, actually generate report in test plus other tests to make it to the
codecov target

JordiBForgeFlow committed:
[13.0][fix][account_asset_management] display only accounts associated
to the current company.

JordiBForgeFlow committed:
[13.0][account_asset_management] button to navigate from the vendor bill
to the asset.

JordiBForgeFlow committed:
[13.0][account_asset_management] propose account when user chooses

MateuGForgeFlow committed:
[13.0][account_asset_management] propose account when user chooses

victoralmau committed:
[FIX] account_asset_management: Fix onchange account_id and asset_profile_id to prevent account_id is empty in some use cases

victoralmau committed:
[FIX] account_asset_management: Prevent create lines with init=True when account_lock_date_update addon is installed and lock date defined

Yvesldff committed:
Translated using Weblate (French)

Currently translated at 95.8% (185 of 193 strings)

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/fr/

yelizariev committed:
[FIX] account_asset_management: correct field name

joao-p-marques committed:
[IMP] account_asset_management: Add analytic tags and propagate

TT28974

Yvesldff committed:
Translated using Weblate (French (France))

Currently translated at 97.9% (189 of 193 strings)

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/fr_FR/

JordiMForgeFlow committed:
account_asset_management: fix group_assets sorting

JordiMForgeFlow committed:
[13.0][account_asset_management]: add user error when creating asset from bill

StefanRijnhart committed:
[FIX] account_asset_mamagement: set invoice date

Adapt to https://github.com/odoo/odoo/commit/fcaa54939e, which prevents
setting a default invoice date for purchase documents

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 66.3% (128 of 193 strings)

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/nl/

MiquelRForgeFlow committed:
[FIX] account_asset_management: include 'number' method_time

MiquelRForgeFlow committed:
[FIX] account_asset_management: create asset using Form

This way, the computes are computed as expected.

oca-travis committed:
[UPD] Update account_asset_management.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/

victoralmau committed:
[FIX] account_asset_management: Fix error from group_ids in asset when set profile_id

victoralmau committed:
[FIX] account_asset_management: Fix error according to _name_search in group when try to set groups in profile

JordiMForgeFlow committed:
[13.0][FIX]account_asset_management: use column_invisible for fields inside a tree of a o2m field

JordiMForgeFlow committed:
[13.0][IMP]account_asset_management: add optional attribute for added columns in invoice lines

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/

Yvesldff committed:
Translated using Weblate (French)

Currently translated at 99.0% (218 of 220 strings)

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/fr/

JordiBForgeFlow committed:
[IMP] account_asset_management: add carry_forward_missed_depreciations

JordiBForgeFlow committed:
[IMP] account_asset_management: Allow to reverse the posting of a depreciation line
instead of deleting the journal entry.
This will be done when the company has activated the inalterability hash
on the original journal entry.

ps-tubtim committed:
[14.0][FIX] account_asset_management: pre-commit

pedrobaeza committed:
[FIX] account_asset_management: Don't put asset name as entry name

This can provoke an error due to duplicated entry name.

Remaining of the migration to version 13.0, and similar to
https://github.com/OCA/account-financial-tools/commit/b7637cbe45701700148efd6d9e32fb0c0079b4f2#diff-5318cd3560cdb2d23d2889fe39c5b71745c140810b20f64f0c9aa5b24b8b602bL206

Commit:
[FIX] account_asset_management: make asset company match move's

To start, the context `company_id` is no longer used in v14. Instead, we use `with_company`.

OTOH, it doesn't make much sense because in case you have several companies enabled in your context, not necessarily your main one is gonna be the one that should be used for the asset. Instead, it should be the move company always, or consistency rules will fail.

Altered forward port of https://github.com/OCA/account-financial-tools/pull/1223.

@Tecnativa TT31311

ps-tubtim committed:
[FIX] account_asset_management: fix _check_dates

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/

kittiu committed:
[14.0][FIX] account_asset_management, error on remove for asset no depre

kittiu committed:
[14.0][FIX] account_asset_management, fix multi currency

ps-tubtim committed:
[ENH] account_asset_management: Add purchase value in asset report

MiquelRForgeFlow committed:
[IMP] account_asset_management: hook for asset vals

kittiu committed:
[14.0][FIX] account_asset_management, singleton error on asset_count

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/

kittiu committed:
[14.0][FIX] account_asset_management, error create bill by purchase user

alexis-via committed:
account_asset_management: improve and cleanup views

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/

victoralmau committed:
[FIX] account_asset_management: Avoid the error when clicking on "Delete" button if there is a residual value and all the depreciation lines are posted.
TT32861

ps-tubtim committed:
[FIX] account_asset_management: sudo smart button asset

RachidAlassir committed:
Translated using Weblate (Arabic)

Currently translated at 34.5% (75 of 217 strings)

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/ar/

etobella committed:
[UPD] account_asset_management: Mark as mature

sbidoul committed:
Bump account_asset_management version

JordiBForgeFlow committed:
[IMP] account_asset_management: assets can be removed irrespective of the Time Method used
It does not make sense that you cannot remove an asset where the time method used
is not based on number of years or end date. If you choose, for example, to depreciate the asset
using the time method 'Number of depreciations' you will want to remove the asset when it has been
fully depreciated.

Saran440 committed:
[FIX] calculate day mistake, remove asset

mariadforgeflow committed:
[IMP] account_asset_management: set asset_profile_id on invoice creation

victoralmau committed:
[IMP] account_asset_management: Add chatter + activities to assets.
TT34459

oca-travis committed:
[UPD] Update account_asset_management.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/

OCA-git-bot committed:
[IMP] update dotfiles

mariadforgeflow committed:
[IMP] account_asset_management: prevent from creating assets from tax lines

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 26.9% (71 of 263 strings)

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_asset_management/it/

manuelregidor committed:
[IMP] : black, isort, prettier

manuelregidor committed:
[MIG]account_asset_management: Migration to 15.0

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (263 of 263 strings)

Translation: account-financial-tools-15.0/account-financial-tools-15.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-15-0/account-financial-tools-15-0-account_asset_management/es_AR/

Commit:
[15.0][FIX] account_asset_management: add post-install tagged.

LudLaf committed:
Translated using Weblate (Spanish)

Currently translated at 98.8% (260 of 263 strings)

Translation: account-financial-tools-15.0/account-financial-tools-15.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-15-0/account-financial-tools-15-0-account_asset_management/es/

Commit:
[MIG] account_asset_management_menu: Migration to 15.0

Saran440 committed:
[FIX] account_asset_management: Allow to write on moves with no asset permission

Steps to reproduce the problem:

- User (Not accountant) create an invoice.
- Create invoice plan with Deposit on 1st Invoice
- Confirm Order > Register Deposit > Create and View bills
- It throws a permission error

That's because the search on asset lines is done always for each write on the account.move
if certain fields (like the date) are written.

OCA-git-bot committed:
[UPD] README.rst

KNVx committed:
[FIX] account_asset_management: Poor performance updating assets massively

When updating assets massively the performance when searching for deprecation_lines_ids is very low.
It is a difficult problem to detect since massive asset updates are not common.
Indexing in this field dramatically improves the timing performance.
For example, to update 3000 records it took 30 sec. With this index it takes to 2 seconds.

alexis-via committed:
[IMP] : pre-commit stuff

RodrigoBM committed:
[MIG] account_asset_management: Migration to 16.0

RodrigoBM committed:
[REF] account_asset_management: the name of the company_currency_id field is changed to currency_id and currency_field is not required.

ps-tubtim committed:
[ENH] account_asset_management: default salvage value on profile

This PR enables the configuration of salvage value in the asset profile.
Users can configure it with a fixed amount or a percentage of the salvage value.

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 85.1% (224 of 263 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/hr/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 26.9% (71 of 263 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/it/

sysadminmatmoz committed:
Translated using Weblate (Slovenian)

Currently translated at 52.0% (137 of 263 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/sl/

sanrav committed:
Translated using Weblate (French (France) (fr_FR))

Currently translated at 100.0% (263 of 263 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/fr_FR/

sanrav committed:
Translated using Weblate (French)

Currently translated at 80.2% (211 of 263 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/fr/

Commit:
[UPD] Update account_asset_management.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/

badbole committed:
Translated using Weblate (Croatian)

Currently translated at 92.6% (240 of 259 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/hr/

sanrav committed:
Translated using Weblate (French)

Currently translated at 89.5% (232 of 259 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/fr/

Gelo-fl committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (259 of 259 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/es/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 76.0% (197 of 259 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/pt_BR/

marcelsavegnago committed:
[IMP] account_asset_management: add asset_profile_id and asset_id on invoice_lines form

jdidderen committed:
[FIX] account_asset_management - use company from env and not user

RodrigoBM committed:
[FIX] account_asset_management: pass for context correct active_id. If not specified the value of active_id will be that of account.asset.line.

kikopeiro committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (259 of 259 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/es/

Commit:
[UPD] Update account_asset_management.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (259 of 259 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/es/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 27.0% (70 of 259 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/it/

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 27.1% (70 of 258 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/it/

newtratip committed:
[16.0][FIX] account_asset_management: use Reversal Date to Accounting Date of reversal entry

Commit:
Translated using Weblate (Dutch)

Currently translated at 46.1% (119 of 258 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/nl/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (258 of 258 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/fr/

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (258 of 258 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/es_AR/

ThijsvOers committed:
Translated using Weblate (Dutch)

Currently translated at 87.2% (225 of 258 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/nl/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 89.1% (230 of 258 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_asset_management/pt_BR/

sbiosca-s73 committed:
[IMP] account_asset_management: pre-commit auto fixes

sbiosca-s73 committed:
[MIG] account_asset_management: Migration to 17.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-17.0/account-financial-tools-17.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-17-0/account-financial-tools-17-0-account_asset_management/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 42.0% (108 of 257 strings)

Translation: account-financial-tools-17.0/account-financial-tools-17.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-17-0/account-financial-tools-17-0-account_asset_management/it/

pedrobaeza committed:
[FIX] account_asset_management: Allow to unlink w/o billing permission

Some users may be allowed to unlink invoices without billing access,
but with current code, they are not able, as the asset line check is
done without sudo, and other users except billing or higher ones
aren't allowed to access to that model.

TT49673

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-17.0/account-financial-tools-17.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-17-0/account-financial-tools-17-0-account_asset_management/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (256 of 256 strings)

Translation: account-financial-tools-17.0/account-financial-tools-17.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-17-0/account-financial-tools-17-0-account_asset_management/it/

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (256 of 256 strings)

Translation: account-financial-tools-17.0/account-financial-tools-17.0-account_asset_management
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-17-0/account-financial-tools-17-0-account_asset_management/es_AR/

luc-demeyer committed:
[FIX] account_asset_management: fix asset report

The Financial Assets report generates an error message where there is an asset
without depreciation table.
This is ok for most assets but not for those with method_number = 0 which
is the case for assets of type 'Land and Buildings'.

carlos-lopez-tecnativa committed:
[IMP] account_asset_management: pre-commit auto fixes

carlos-lopez-tecnativa committed:
[MIG] account_asset_management: Migration to version 18.0

carlos-lopez-tecnativa committed:
[FIX] account_journal_restrict_mode: Make compatible with other modules

When this module is installed, it prevents the cancellation of posted journals. However, in many cases, this action is necessary—for example, in the account_asset_management module.

With this commit, the default constraint is applied only in this module's tests or when Odoo is not running in test mode.

This approach provides an alternative to avoid marking the module as rebellious.

pedrobaeza commented:
/ocabot migration account_asset_management

victoralmau reviewed (approved):
Code and functional review OK.

juancarlosonate-tecnativa commented:
Update when you can, https://github.com/OCA/reporting-engine/pull/958 has already been merged, thx!

carlos-lopez-tecnativa commented:
> Update when you can, [OCA/reporting-engine#958](https://github.com/OCA/reporting-engine/pull/958) has already been merged, thx!

Done. I rebased and removed the last commit. Thanks!

carlos-lopez-tecnativa commented:
@juancarlosonate-tecnativa could you please update your review?

Saran440 commented:
Functional Test

1. please add this commit https://github.com/OCA/account-financial-tools/pull/1820
2. Asset Report still mistake when we are multi-company. it still export all data https://github.com/OCA/account-financial-tools/pull/1912

carlos-lopez-tecnativa commented:
> Functional Test
> 
> 1. please add this commit [[16.0][ENH] account_asset_management: default salvage value on profile #1820](https://github.com/OCA/account-financial-tools/pull/1820)

I created a PR for V17 to include it as well. Once [this PR](https://github.com/OCA/account-financial-tools/pull/2047) is merged, I will cherry-pick it to V18. Thanks.

> 2. Asset Report still mistake when we are multi-company. it still export all data [[15.0][FIX] account_asset_management: report filter multi company #1912](https://github.com/OCA/account-financial-tools/pull/1912)

Regarding the second point, I cannot include a PR that is still open and has pending review comments. Once it is merged, it can be forward-ported to V16, V17, and V18 (if merged before, or feel free to ping me to include it).

Saran440 commented:
@carlos-lopez-tecnativa Thanks you for your work.

I am waiting for the review after you cherry-pick https://github.com/OCA/account-financial-tools/pull/2047 into this.

carlos-lopez-tecnativa commented:
> @carlos-lopez-tecnativa Thanks you for your work.
> 
> I am waiting for the review after you cherry-pick #2047 into this.

Done, please update your review, thanks

Saran440 reviewed (approved):
Functional Test :+1:

pedrobaeza commented:
/ocabot merge nobump
